treeherder/model/derived/jobs.py

```diff
@@ -731,30 +723,27 @@ def get_result_set_details(self, result_set_ids):
 
         return aggregate_details
 
-    def set_lower_tier_signatures(self):
+    def _get_lower_tier_signatures(self):
         # get the lower tier data signatures for this project.
         # if there are none, then just return an empty list
         # this keeps track of them order (2, then 3) so that the latest
         # will have precedence.  If a job signature is in both Tier-2 and
         # Tier-3, then it will end up in Tier-3.
-        self.lower_tier_signatures = []
+        lower_tier_signatures = []
         for tier_num in self.LOWER_TIERS:
```

Comment:
Perhaps you could just reference the ``ExclusionProfileManager`` LOWER_TIERS value here instead of keeping one local to ``jobs.py``.